### PR TITLE
14.0 [MIG] maintenance_equipment_tags

### PR DESCRIPTION
**Pull request by mariadforgeflow**
Standard migration to v14 @MiquelRForgeFlow @GenisPForgeFlow

### CODE REVIEW

**Review by GenisPForgeFlow — APPROVED** [review]
LGTM!

File: maintenance_equipment_tags/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Maintenance Equipment Tags",
+    "summary": """
+        Adds category tags to equipment""",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Creu Blanca,Odoo Community Association (OCA)",
```
Comment (MiquelRForgeFlow) [inline maintenance_equipment_tags/__manifest__.py:10]:
```suggestion
    "author": "Creu Blanca,Odoo Community Association (OCA)",
    "maintainers": ["jaimearroyo7"],
```
I think you could add this

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by etobella — APPROVED** [review]
LGTM no major changes

File: maintenance_equipment_tags/__manifest__.py (hunk 1 above)
Comment (mariadforgeflow) [reply maintenance_equipment_tags/__manifest__.py:10]: Done!

Comment (MiquelRForgeFlow) [reply maintenance_equipment_tags/__manifest__.py:10]: then mark as resolved this comment :)

Comment (etobella) [reply maintenance_equipment_tags/__manifest__.py:10]: Put me please as maintainer, Jaime is no longer working with odoo

**Review by etobella — APPROVED** [review]

**Pull request merged**